### PR TITLE
Adding documentation for the LM layer modifier.

### PR DESCRIPTION
**Pull request by davidbfrogman**
This will hopefully clear up some of the confusion around adding this layer modifier.  I struggled with this a bit as well at first.

Let me know if there's an easier way to format this.

### CODE REVIEW

**Comment by davidbfrogman** [comment]
Also hopefully helps close out #3185.  I'm happy to add this in other places as well.

**Comment by davidbfrogman** [comment]
Also not sure why that table is there 2 times in this documentation here.  I'm happy to delete the duplicate.

**Comment by fauxpark** [comment]
The second table is the one to keep - perhaps it could be moved up to where the first one is. The list of modifiers is actually documented [here](https://docs.qmk.fm/#/feature_advanced_keycodes?id=mod-tap) but they should really have their own section rather than seeming MT specific.

I also think the keycode descriptions should not include examples or details, just a brief summary of what the keycode does. That's what the pages linked in the headers are for. But that's up to someone with a merge button :P

File: docs/keycodes.md [hunk 1]
```diff
@@ -282,17 +282,17 @@ This is a reference only. Each group of keys links to the page documenting their
...
-|Key         |Description                                                               |
-|-----------------|---------------------------------------------------------------------|
-|`DF(layer)`      |Switches the default layer                                           |
-|`MO(layer)`      |Momentarily activates layer, switches off when you let go            |
-|`LM(layer, mod)` |As `MO(layer)` but with `mod` active                                 |
-|`LT(layer, kc)`  |Momentarily activates layer if held, sends kc if tapped              |
-|`TG(layer)`      |Toggles the layer (enables it if no active, and vise versa)          |
-|`TO(layer)`      |Activates layer and deactivates all other layers                     |
-|`TT(layer)`      |Momentarily activates layer if held, toggles it if tapped repeatedly |
+## [Switching and Toggling Layers](feature_advanced_keycodes.md#switching-and-toggling-layers)
```
Comment (drashna) [inline docs/keycodes.md]: "Changing Layers" may be a better (shorter) phrase here.

**Comment by davidbfrogman** [comment]
Changed to suggested title.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**